patches/chromium/gpu_notify_when_dxdiag_request_fails.patch

```diff
@@ -0,0 +1,69 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Deepak Mohan <hop2deep@gmail.com>
+Date: Fri, 28 Feb 2020 15:08:26 -0800
+Subject: gpu: notify when dxdiag request for gpu info fails
+
+We rely on the signal OnGpuInfoUpdate to process gpu info.
```

Comment:
The spec `app module getGPUInfo() API succeeds with complete GPUInfo` failure on windows in this chromium roll revealed two issues:

1) The complete gpu info involves requesting the gpu process for info about DxDiagnostics and Dx12Vulkan specific infos, but our test was not giving enough time to launch the gpu process and thus only obtained basic info, this is fixed in https://github.com/electron/electron/pull/22198/commits/64aa2fae5e011a8413b44a098da582e9112ef911

2) There is a flaw in the current `getGPUInfo` api where it waits for the above mentioned values from the gpu process https://github.com/electron/electron/blob/master/shell/browser/api/gpuinfo_manager.cc#L37 hoping that it will always succeed https://source.chromium.org/chromium/chromium/src/+/master:content/browser/gpu/gpu_data_manager_impl_private.cc;l=837, but there are situations when this may not succeed https://source.chromium.org/chromium/chromium/src/+/master:content/browser/gpu/gpu_data_manager_impl_private.cc;l=630-647 in which case the api just waits endlessly, this patch fixes that case by reusing the failure signals from chromium. 

If the api was instead an event like `app.on('gpu-info-update')` users can continuously perceive the updates from gpu process and we don't have to worry about handling these edge cases similar to `chrome://gpu` page.